### PR TITLE
minor ui fix

### PR DESCRIPTION
**Pull request by git-ruthvik**

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -12,17 +12,29 @@ class Footer extends React.Component {
       <footer className="footer">
         <Container fluid>
           <div className="copyright">
+<<<<<<< Updated upstream
```
Comment (Impeekay) [inline src/components/Footer/Footer.js:15]: wha this this Upstream?? something wrong with the way you've pulled. Check once

Comment (git-ruthvik) [reply src/components/Footer/Footer.js:15]: remove those lines, its from the vscode git changes thing

Comment (git-ruthvik) [reply src/components/Footer/Footer.js]: I have fixed the code now

**Comment by mesrivatsav** [comment]
pull both one after other even ruthwik will get the contribution mine will work i checked u can pull mine also

**Comment by Impeekay** [comment]
> pull both one after other even ruthwik will get the contribution mine will work i checked u can pull mine also

if you check above on the pull-request it says files-changed 0, if that is the case, there will be no contribution reflecting on contributions also

**Comment by mesrivatsav** [comment]
leave let his name be in commits atleast

**Comment by Impeekay** [comment]
> leave let his name be in commits atleast

pull this branch and check if nothing is wrong, if yes let me know

**Comment by mesrivatsav** [comment]
I only sent that code no errors u can pull

**Comment by Impeekay** [comment]
> I only sent that code no errors u can pull

your responsibility!

**Pull request merged**